src/ctapipe/tools/tests/test_calculate_pixel_stats.py

```diff
@@ -50,6 +49,9 @@ def test_calculate_pixel_stats_tool(tmp_path, dl1_image_file):
         argv=[
             f"--input_url={dl1_image_file}",
             f"--output_path={monitoring_file}",
+            "--r0-waveforms",
```

Comment:
is this carrying over R0 waveforms? why do we need it?Following the discussion on slack, by default we should strip everything apart from metadata and only write the results.

Reply:
I think you want to at least copy:

- subarray
- observation block and scheduling block info

and maybe the simulation config in case of simulations

Reply:
from the actual payload data in addition to the metadata.

Reply:
Yes, for now I would hardcode what you need in your usecase

Reply:
I thought having a context manager setting and resetting defaults would work... but it doesn't and I don't understand why yet. All looks good until the instance is created and it gets the wrong default for some reason